pkg/ui/constants.go

```diff
@@ -22,6 +22,7 @@ const (
 	// Page Titles
 	AlertsPageTitle          = "Alerts"
 	AlertDataPageTitle       = "Metadata"
+	AlertMetadata            = "AlertData"
```

Comment:
Why is this needed?

Reply:
When we view the sop data and if we want to return back to triggered incident view , then this helps us to move back from metadata view to back to triggered incident view.